### DISCUSSION TITLE
How to edit saved searches?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [ammargitham/WallFlow#71] [Q&A] [answered]
I have saved some searches, but how can we perform delete/edit/rename on there searches?

**Comment by ammargitham** [comment, marked as answer]
Hi. You can edit them in Settings -> Manage saved searches (Under General category)

**Reply by KAGEYAM4** [reply]
Thank you.


Do you it would be better UX wise, to also add this to main screen page where we select `filter>load` , so after loading a saved-search, the title of filter will show the loaded saved search, after making changes we can use save and it will save the new changes the the loaded search. 
Also if user dosen't use the save button, the filter changed filter setting will still be there even after closing the fiter page.

**Reply by ammargitham** [reply]
Can you open an issue for both of them? I can look into it.
